dvc/command/experiments.py

```diff
@@ -1045,6 +1081,34 @@ def add_parser(subparsers, parent_parser):
             "across the selected experiments."
         ),
     )
+    experiments_show_parser.add_argument(
+        "--html",
+        action="store_true",
+        default=False,
+        help="Generate a parallel coordinates plot from the tabulated output.",
+    )
+    experiments_show_parser.add_argument(
+        "--color-by",
```

Comment:
Really like the idea

Reply:
Also, can we use `--sort-order` if we do this to allow for inverting the scale?

Reply:
Sure

Reply:
About `--sort-by`, I implemented it and it kind of feels good for scalars but it feels a little strange for categorical values (i.e. `--sort-by params.yaml:optimizer`). Perhaps this is a problem derived from integrating the plot under `dvc exp show`; If we had a separate command, having a specific `--color-by` option would be more intuitive but I guess that the motivation here is to reduce the number of available options.

About `--sort-order`, what would you expect to be inverted? 
We don't currently expose the colorbar selection, would you expect to be the colors inverted or the ticks of the colorscale on the side?

Reply:
> feels a little strange for categorical values

Is this any different for the regular table view? It's an awkward sort column in either case.

> having a specific --color-by option would be more intuitive

Do you mean it's useful to color by a categorical column, but sorting by it is awkward? I think if I am coloring by that column, I still want a predictable order for it. Maybe I need to see an example to understand the problem.

> About --sort-order, what would you expect to be inverted?

If you invert the ticks, won't the colors of the individual lines be inverted? That's what I would expect.

Reply:
> Is this any different for the regular table view? It's an awkward sort column in either case.

In table it's an awkward sort but in plot, it is that + the option that defines the color so it makes it a little more strange.

> Do you mean it's useful to color by a categorical column, but sorting by it is awkward? I think if I am coloring by that column, I still want a predictable order for it. Maybe I need to see an example to understand the problem.

I just mean that the name is strange, sometimes I would just use a categorical column to color, don't caring about the order and the option being named `--sort-by` is strange.

> If you invert the ticks, won't the colors of the individual lines be inverted? That's what I would expect.

Ticks and colorscale are decoupled in plotly that's why I'm not sure what effect would be expected when using `--sort-order` in the plot.

Reply:
I see what you mean about the awkwardness of `--sort-by`. I would still vote to use it since otherwise we need a flag specific to this output type, plus we need to figure out how that should interact with `--sort-by` and `--sort-order` here.

For `--sort-order`, I would invert the ticks.

Reply:
`--sort-by` is already pushed. working on `--sort-order`

Reply:
Resolving. `--sort-order` won't be included as part of this P.R. Opened https://github.com/iterative/dvc/issues/7081 to track it